### PR TITLE
Added support for bulk indexing on Blazegraph

### PR DESCRIPTION
**Pull request by umbreak**
Fixes https://github.com/BlueBrain/nexus/issues/451

### CODE REVIEW

File: src/main/resources/contexts/resource-context.json [hunk 1]
```diff
@@ -1,46 +1,16 @@
...
-    },
-    "SparqlView": {
-      "@id": "nxv:SparqlView",
-      "@type": "@id"
-    },
-    "Alpha": {
-      "@id": "nxv:Alpha",
-      "@type": "@id"
-    },
+    "Schema": "nxv:Schema",
```
Comment (bogdanromanx) [inline src/main/resources/contexts/resource-context.json]: why this change?

File: src/main/scala/ch/epfl/bluebrain/nexus/kg/config/AppConfig.scala [hunk 2]
```diff
@@ -189,6 +189,17 @@ object AppConfig {
...
   final case class IndexingConfig(batch: Int, batchTimeout: FiniteDuration, retry: RetryStrategyConfig)
 
+  /**
+    * The indexing configuration for the different indexers
+    *
+    * @param sparql        the indexing configuration applied to Sparql indexer
+    * @param elasticSearch the indexing configuration applied to Elastic Search indexer
+    * @param keyValueStore the indexing configuration applied to the Distributed Data store
+    */
+  final case class IndexingConfigCollection(sparql: IndexingConfig,
```
Comment (bogdanromanx) [inline src/main/scala/ch/epfl/bluebrain/nexus/kg/config/AppConfig.scala]: rename to `IndexingConfigs` ?

File: src/main/resources/contexts/resource-context.json (hunk 1 above)
Comment (umbreak) [reply src/main/resources/contexts/resource-context.json]: Because it is not needed. Those aliases are for things like
```json
{
   "@type": "Schema"
}
```
So an alias is enough, as you can check with the playground.
